docs/source/driver/go/flight_sql.rst

```diff
@@ -130,11 +130,16 @@ These options map 1:1 with the options in FlightClientOptions:
     Override the root certificates used to validate the server's TLS
     certificate.
 
-``arrow.flight.sql.client_option.generic_int_option.<OPTION_NAME>``
-    Option prefixes used to specify generic transport-layer options.
-
-``arrow.flight.sql.client_option.generic_string_option.<OPTION_NAME>``
-    Option prefixes used to specify generic transport-layer options.
+``arrow.flight.sql.client_option.with_block``
+    Whether connections should wait until connections are established,
+    or connect lazily when used.  The latter is gRPC's default
+    behavior, but the driver defaults to eager connection to surface
+    errors earlier.  Value should be ``true`` or ``false``.
+
+``arrow.flight.sql.client_option.with_max_msg_size``
+    The maximum message size to accept from the server.  The driver
+    defaults to 16 MiB since Flight services tend to return larger
+    reponse payloads.  Should be a positive integer number of bytes.
```

Comment:
we have a slight disconnect.... the docs say `arrow.flight.sql.....` but the actual flags we're using are `adbc.flight.sql....` which should it actually be? we should update the docs/code/tests so everything is in sync with what is expected here.

Reply:
`adbc` is fine, didn't realize - these are the old docs for C++ that I just ported over